### PR TITLE
First milestone

### PR DESCRIPTION
**Pull request by vitorelourenco**
Navbar added for all screen sizes
Footer added for all screen sizes
Bootstrap was used to the best of our ability, given the content available on day one for the first milestone.

### CODE REVIEW

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUESTED🛑🛑🛑🛑**
Hello team,
Great work on this project👏👏👏. It looks the same as the original.
However let's make some changes to make the project impeccable.

[REQUIRED]: Remove the bottom scroll bar on the navbar.
[REQUIRED]: The navbar should be at the top and the width should be 100%
[REQUIRED]: Kindly use more bootstrap classes wherever possible. for example the .logo in css can have d-flex instead of putting it in css.
[OPTIONAL]: Use the color picker chrome extension to pick colors from the original site.

Kindly make the required changes and ask for another review👌
You got this👍
Happy coding💻

**Comment by Amalcxc** [comment]
original
Screen Shot 2021-03-02 at 5 56 24 PM
clone
Screen Shot 2021-03-02 at 5 56 09 PM
it is not 100% width in the original as you can see.

**Comment by vitorelourenco** [comment]
The bottom scroll bar was removed
The navbar was placed at the top
Instances of display: flex were replaced with class="d-flex"
The shade of red used in the paged was corrected to match the original one

File: compiled_css/styles.css [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+}
+
+.space {
+  min-height: 65px;
+  background: none transparent;
+}
+
+.page-header {
+  display: none;
+}
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:21]: - [ ] This can be done with the bootstrap class `d-none`

File: compiled_css/styles.css [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+  width: 30px;
+  height: 30px;
+  padding-left: 0;
+  opacity: 0.75;
+  border: none;
+}
+
+.pelcro-subscribe-button {
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:41]: - [ ] This can be done with the bootstrap class `text-uppercase`

File: compiled_css/styles.css [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+}
+
+.page-footer li {
+  margin-left: 10px;
+  list-style: none;
+}
+
+.page-footer ul {
+  padding: 0;
+}
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:73]: - [ ] This can be done with the bootstrap class `p-0`

File: compiled_css/styles.css [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+
+.page-footer p {
+  font-size: 12px;
+  margin-top: 2px;
+  margin-bottom: 0;
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:78]: - [ ] This can be done with the bootstrap class `mb-0`

File: compiled_css/styles.css [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+}
+
+.page-footer a {
+  text-decoration: none;
+  color: gray;
+}
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:84]: - [ ] This can be done using bootstrap classes

File: compiled_css/styles.css [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+  font-size: 24px;
+}
+
+.footer-links ul {
+  flex-flow: row wrap;
+  margin-bottom: 0;
+}
+
+.footer-links li {
+  display: inline-block;
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:102]: - [ ] This can be done with the bootstrap class `d-inline-block`

File: compiled_css/styles.css [hunk 7]
```diff
@@ -0,0 +1,323 @@
...
+  cursor: pointer;
+}
+
+.footer-links {
+  border-top: 1px solid grey;
+}
+
+.footer-links-menu li {
+  margin-left: 0;
+}
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:114]: - [ ] This can be done with bootstrap classes

File: compiled_css/styles.css [hunk 8]
```diff
@@ -0,0 +1,323 @@
...
+    max-width: 1320px;
+  }
+
+  /* .logo {
+    display: flex;
+  }
+
+  .logo-p {
+    display: flex;
+  } */
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:132]: - [ ] Please remove commented code. It is bad practice to leave commented code in your files, if you need to see a previous version of your code, you can check previous commits.

File: compiled_css/styles.css [hunk 9]
```diff
@@ -0,0 +1,323 @@
...
+@media (min-width: 988px) {
+  .top-buttom-wrapper {
+    align-items: center;
+    height: 30px;
+  }
+
+  /* .jc-sb {
+    justify-content: space-between;
+    align-items: flex-end;
+  } */
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:169]: - [ ] Please remove commented code.

File: compiled_css/styles.css [hunk 10]
```diff
@@ -0,0 +1,323 @@
...
+    background: #f72210;
+    display: none;
+  }
+
+  .trending-topics-wrapper ul {
+    /* display: flex; */
+    overflow-x: hidden;
+
+    /* overflow-x: scroll;
+    overflow-y: hidden; */
```
Comment (BrittanyBlake) [inline compiled_css/styles.css:181]: - [ ] Please remove commented code.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,199 @@
...
+            <button class="pelcro-login-button login">LOGIN</button>
+            <button class="pelcro-subscribe-button align-items-start">Subscribe</button>
+            <i class="fas fa-bars"></i>
+          </div>
+          
+        </header>
+      </div>
+    </div>
+
+<footer class="page-footer container-fluid">
```
Comment (BrittanyBlake) [inline index.html]: - [ ] Your footer should stick to the bottom of the page even if there is no content. You can make your footer stick to the bottom by adding the following bootstrap class `fixed-bottom`. 

```suggestion
    <footer class='page-footer container-fluid fixed-bottom'>
```

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi Team! 👋 
You have done a great job working on the changes requested by the previous TSE👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] There are still a lot of css styles can be done with bootstrap classes. Take a look at your css file again and use this [cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) to see what can be done with bootstrap instead. 
- [ ] Your footer should stick to the bottom of the page even if there is no content.
- [ ] You have a lot of instances of commented code in your css file. Please remove them as it is bad practice.

****

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by vitorelourenco** [comment]
Fixed all the issues pointed out by BrittanyBlake

**Comment by vitorelourenco** [comment]
Please take into consideration that the middle part of the page is not a part of mielstone1, it is there because we are working on it in between code reviews for this first milestone

File: README.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# The Next Web Home Page
```
Comment (BrittanyBlake) [inline README.md:3]: - [ ] This should not say The Next Web. Please update your readme to reflect the current project.

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**